src/pages/main-page/main-page.js

```diff
@@ -0,0 +1,161 @@
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography
+} from '@material-ui/core';
+import { getRecentComments } from '../../redux/comments/comments.actions';
+import { getOrderList } from '../../redux/orders/orders.actions';
+import LoadingBar from '../../components/loading-bar';
+import titles from '../../configs/titles';
+
+import { useCommonStyles } from '../common.styles';
+import { useStyles } from './main-page.styles';
+
+const MainPage = () => {
+  const {
+    mainTitle,
+    commentsTitle,
+    ordersTitle,
+    changesTitle
+  } = titles.mainPageTitles;
+  const classes = useStyles();
+  const commonClasses = useCommonStyles();
+  const dispatch = useDispatch();
+  const { commentsList, commentsLoading } = useSelector(({ Comments }) => ({
+    commentsList: Comments.list,
+    commentsLoading: Comments.commentsLoading
+  }));
+
+  const { orderLoading, ordersList } = useSelector(({ Orders }) => ({
+    orderLoading: Orders.orderLoading,
+    ordersList: Orders.list.items
+  }));
+
+  useEffect(() => {
+    dispatch(getRecentComments());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(
+      getOrderList({
+        filter: {
+          orderStatus: 'CREATED'
+        }
+      })
+    );
+  }, [dispatch]);
+
+  const comments = commentsList.map(({ date, text, user, _id }) => (
+    <div key={_id} className={classes.comment}>
+      <div className={classes.commentText}>{text}</div>
+      <div className={classes.commentInfo}>
+        <div>{user.name || 'Гість'}</div>
+        <div>{moment.unix(date / 1000).format('HH:mm DD.MM.YYYY ')}</div>
+      </div>
+    </div>
+  ));
+
+  if (orderLoading || commentsLoading) {
+    return <LoadingBar />;
+  }
+
+  return (
+    <div className={`${commonClasses.container} ${classes.root}`}>
+      <Typography
+        variant='h3'
+        className={commonClasses.materialTitle}
+        data-cy='page-title'
+      >
+        {mainTitle}
+      </Typography>
+      <div className={classes.main}>
+        <div className={classes.commentsOrders}>
+          <Paper className={classes.ordersContainer} data-cy='orders-container'>
+            <Typography variant='h5' className={classes.blockTitle}>
+              {ordersTitle}
+            </Typography>
+            {ordersList && ordersList.length ? (
+              <TableContainer className={classes.orders}>
+                <Table
+                  stickyHeader
+                  aria-label='sticky table'
+                  data-cy='orders-table'
+                >
+                  <TableHead>
+                    <TableRow>
+                      <TableCell component='th' align='center'>
+                        Дата
+                      </TableCell>
+                      <TableCell component='th' align='center'>
+                        Сума
+                      </TableCell>
+                      <TableCell component='th' align='center'>
+                        ID замовлення
+                      </TableCell>
```

Comment:
Move this hardcode to the config file
We use component TableContainerGenerator for all tables in our project maybe you should use it here too

Reply:
fixed